### DISCUSSION TITLE
sonoff pow origin/elite powr320D spontaneous boot loop when flashed

### DISCUSSION THREAD

**Discussion by bryanb334** [arendst/Tasmota#19979] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
After successfully flashing sonoff pow origin/elite powr320D it enters a boot loop, just constantly rebooting before it gets a chance to start the AP.
I have flashed with both espflasher and the web flasher, both with the same results.
I have tried the current version of Tasmota32.bin (13.2.0) and an older version (12.5.0) with the same result.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____
- [x] Tasmota binary firmware version number used: _____
  - [x] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
It should start an access point so I can enter wifi details etc.

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by bryanb334** [comment]
unable to run commands ....

**Comment by sfromis** [comment, marked as answer]
You posted no details about what happens, like the log. Still, typical issue is that it will not boot to Wifi without using mains power instead of the setup for flashing.

**Comment by bryanb334** [comment]
What I am getting is constant rebooting so not time for the wifi access
point to establish.  When I connect to mains, nothing happens.  There are
no LEDs, no AP.  Here is a few seconds of the logs:


rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.037-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.059 CFG: Loaded from File, Count 2
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002 HDW: ESP32-D0WD-V3
00:00:00.040 UFS: FlashFS mounted with 312 kB free
00:00:00.045 TFS: File '.settings' not found
00:00:00.046 CFG: Loaded from NVS, Count 2
00:00:00.048 FRC: Some settings have been reset (2)
00:00:00.420 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.441 TFS: File '.drvset003' not found
00:00:00.441 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.449 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.786 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/52 HDW: ESP32-D0WD-V3
00:00:00.040-218/51 UFS: FlashFS mounted with 312 kB free
00:00:00.045-218/51 TFS: File '.settings' not found
00:00:00.051-218/51 TFS: File '.settings.lkg' not found
00:00:00.051-218/51 CFG: Use defaults
00:00:00.240 QPC: Reset
00:00:00.625 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.647 TFS: File '.drvset003' not found
00:00:00.647 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.659 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.136 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.012 CFG: Loaded from File, Count 2
00:00:00.022 QPC: Count 1
00:00:00.358 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.367 TFS: File '.drvset003' not found
00:00:00.367 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.375 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.535 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 2
00:00:00.359 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.368 TFS: File '.drvset003' not found
00:00:00.368 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.376 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.536 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 3
00:00:00.359 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.368 TFS: File '.drvset003' not found
00:00:00.368 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.376 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.536 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 4
00:00:00.359 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.368 TFS: File '.drvset003' not found
00:00:00.368 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.376 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.536 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASHvBOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 5
00:00:00.359 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.368 TFS: File '.drvset003' not found
00:00:00.368 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.376 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.536 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 6
00:00:00.359 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.368 TFS: File '.drvset003' not found
00:00:00.368 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.376 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.536 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.007-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002 HDW: ESP32-D0WD-V3
00:00:00.010 UFS: FlashFS mounted with 312 kB free
00:00:00.013 TFS: File '.settings' not found
00:00:00.013 CFG: Loaded from NVS, Count 2
00:00:00.015 FRC: Some settings have been reset (2)
00:00:00.365 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.376 TFS: File '.drvset003' not found
00:00:00.377 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.385 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.493 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/52 HDW: ESP32-D0WD-V3
00:00:00.011-218/51 UFS: FlashFS mounted with 312 kB free
00:00:00.013-218/51 TFS: File '.settings' not found
00:00:00.016-218/51 TFS: File '.settings.lkg' not found
00:00:00.016-218/51 CFG: Use defaults
00:00:00.194 QPC: Reset
00:00:00.540 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.553 TFS: File '.drvset003' not found
00:00:00.553 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.565 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.853 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.017-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.029 CFG: Loaded from File, Count 2
00:00:00.039 QPC: Count 1
00:00:00.391 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.404 TFS: File '.drvset003' not found
00:00:00.405 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.412 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.520 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.017-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.030 CFG: Loaded from File, Count 2
00:00:00.040 QPC: Count 2
00:00:00.392 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.405 TFS: File '.drvset003' not found
00:00:00.406 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.413 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.521 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.017-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.030 CFG: Loaded from File, Count 2
00:00:00.040 QPC: Count 3
00:00:00.392 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.405 TFS: File '.drvset003' not found
00:00:00.406 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.413 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.521 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4

00:00:00.002-220/51 HDW: ESP32-D0WD-V3
00:00:00.017-218/51 UFS: FlashFS mounted with 308 kB free
00:00:00.030 CFG: Loaded from File, Count 2
00:00:00.040 QPC: Count 4
00:00:00.392 BRY: Berry initialized, RAM used=4711 bytes
00:00:00.405 TFS: File '.drvset003' not found
00:00:00.406 CFG: Energy use defaults as file system not ready or file not
found
00:00:00.413 Project tasmota - Tasmota Version
12.5.0(tasmota32)-2_0_7(2023-04-17T08:07:16)
00:00:00.521 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

**Comment by sfromis** [comment]
Try doing as I suggested. Mains power instead of having the flashing setup connected.

**Comment by gampam2000** [comment]
Any Updates of this issue? I have the exact same problem with a Sonoff POWR320 and POWR316.

I erased the flash and than flashed:

Flash ID Output:
Chip is ESP32-D0WD-V3 (revision v3.1)
Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
Crystal is 40MHz
MAC: 24:dc:c3:a0:ea:0c
Uploading stub...
Running stub...
Stub running...
Manufacturer: 5e
Device: 4016
Detected flash size: 4MB
Hard resetting via RTS pin...

esptool.py --port /dev/ttyUSB0 erase_flash

Flash new Firmware:
esptool.py --port /dev/ttyUSB0 write_flash 0x0 tasmota32.factory.bin

Connected to mains power but nothing happens.
Terminal Output suggest a reboot loop:

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12732
ho 0 tail 12 room 4
load:0x40080400,len:2908
entry 0x400805c4

00:00:00.003-215/50 HDW: ESP32-D0WD-V3 v3.1 
00:00:00.007-213/50 UFS: FlashFS mounted with 308 kB free
00:00:00.013 CFG: Loaded from File, Count 2
00:00:00.023 QPC: Count 6
00:00:00.428 BRY: Berry initialized, RAM used 5164 bytes
00:00:00.443 Project tasmota - Tasmota Version 13.3.0(tasmota32)-2_0_14(2023-12-12T14:31:42)
00:00:00.497 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12732
ho 0 tail 12 room 4
load:0x40080400,len:2908
entry 0x400805c4


Any advice on how to fix this?

**Comment by bryanb334** [comment]
No solution here.  All the suggestions were followed up and tried but did
not help.  The device is still sitting on my desk but will be disposed of
shortly.

**Comment by Jason2866** [comment]
Assemble the device and connect the device to AC mains. Than you can connect to the opened Tasmota AP.
Connect the device to your WiFi.

**Comment by bryanb334** [comment]
You did not read the thread.  I have done that many times.  Flashed many
times. It is outwardly unresponsive when I connect to mains. So no Tasmota
AP shows up.  If I connect back to low voltage (with a good power supply),
I can connect to the serial and it is in a boot loop, spontaneously
rebooting.  Thanks for trying though.

**Comment by gampam2000** [comment]
I finally got it to work:

I used the newest ESP IDF:
esptool.py --port /dev/ttyUSB0 erase_flash

Flash new Firmware:
esptool.py --port /dev/ttyUSB0 write_flash 0x0 tasmota32.factory.bin

After successful flashing, disconnect the Serial and close the case then connected it to 230V Mains (EU-Version).
On my android phone the captive Portal did not show up, but with my pc (linux) it worked. It takes some time to show up after you connected to the AP. I also didn't get any response when I tried to connect to 192.168.4.1

So I quess it is more a problem with the captive portal on ESP32 than the device itself?

**Reply by sfromis** [reply]
In that case, the problem is not with the captive portal, but your phone may have a "security" feature to not "trust" an AP not providing Internet connection, when you connect to it.

Of course this is very different from your posted symptom of
> before it gets a chance to start the AP
